src/EncryptedFS.ts

```diff
@@ -1992,22 +1993,22 @@ class EncryptedFS {
     // It must continue the loop, by restarting the loop with an inherited transaction context
     // This ensures that handling the existing inode is consistent
     let raced = false;
-    let tran: DBTransaction | null = null;
-    let tranRelease: ResourceRelease | null = null;
+    let _tran: DBTransaction | undefined = tran;
+    let _tranRelease: ResourceRelease | undefined = undefined;
```

Comment:
The `_` prefix should be used for parameters that we are not using in a function. Variants of a variable should be `_` suffix.

Additionally, this was previously `null` because `null can be used as an explicit setting, whereas `undefined` can be because a variable was never assigned.

I'd recommend this to continue using `null`. And in that case, you do need to use `tran_` instead of `_tran`.

Same with `_tranRelease`.

Reply:
Fixed this.